CMakeLists.txt

```diff
@@ -35,6 +46,7 @@ endif()
 if(NOT VERSION_NUM)
   include(CMake/git.cmake)
   get_git_tag(VERSION_NUM)
+  string(REGEX MATCH "([0-9]+\.[0-9]+\.[0-9]+)" VERSION_NUM ${VERSION_NUM} )
```

Comment:
I think this might be what is causing the Windows builds to fail.

Reply:
Yes, and no. It fails because `git describe` returns `fatal: No names found, cannot describe anything.`
Before, that resulted only in warning, so the build could continue:
```
CMake Warning at CMakeLists.txt:65 (project):
  VERSION keyword not followed by a value or was followed by a value that
  expanded to nothing.
```